handlers/misttriggers/triggers.go

```diff
@@ -33,6 +33,11 @@ func (d *MistCallbackHandlersCollection) Trigger() httprouter.Handle {
 		}
 
 		triggerName := req.Header.Get("X-Trigger")
+		_ = config.Logger.Log(
+			"msg", "Received Mist Trigger",
```

Comment:
minor: should we log similarly when we ask mist to add PUSH destinations? Might give us a clue as to why we see 2x PUSH_END triggers firing. I'm suspecting we're inadvertently adding an extra PUSH somewhere.

Reply:
Good call, will add in a new PR

Reply:
https://github.com/livepeer/catalyst-api/pull/58